backend/src/main/java/com/google/sps/data/MatchPreference.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** Represent whether or not a Participant is matched */
+public enum MatchPreference {
+  /** Prefer to be matched with different Googler */
+  DIFFERENT(-1),
```

Comment:
In other languages, I think I usually see enum starts with 0, or all with positive numbers, so it's a little bit weird to see negative value here. Is there any reason to assign negative value here?

Reply:
+1 (literally!) please start with 0